modules/appeals_api/app/controllers/appeals_api/notice_of_disagreements/v0/notice_of_disagreements_controller.rb

```diff
@@ -58,6 +88,30 @@ def validate_icn_parameter
       render json: { errors: validation_errors }, status: :unprocessable_entity if validation_errors.present?
     end
 
```

Comment:
(these three methods are equivalent to those in the parent controller, but they don't use an instance variable)